src/index.js

```diff
@@ -1,3 +1,46 @@
 import './index.scss';
 
-console.log('Hello World!');
+const gameId = 'JANudAkE59i9uvoF85MJ';
+
+const user = document.querySelector('.form__element--name');
+const score = document.querySelector('.form__element--score');
+const formSubmitButton = document.querySelector('.form__button');
+const refreshButton = document.querySelector('.main__button');
+const ul = document.querySelector('.ul');
+
+// POST data to API
+formSubmitButton.addEventListener('click', () => {
+  const options = {
+    method: 'POST',
+    body: JSON.stringify({ user: user.value, score: score.value }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  };
+
+  fetch(
+    `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores/`,
+    options,
+  )
+    .then((response) => response.json())
+    .then((data) => console.log(data));
+});
+
+// GET data from API
+async function fetchData() {
+  try {
+    const response = await fetch(
+      `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores/`,
+    );
+    const data = await response.json();
+    for (let i = 0; i < data.result.length; i += 1) {
+      const li = document.createElement('li');
+      li.innerHTML = `<li>${data.result[i].user}: ${data.result[i].score}</li>`;
+      ul.appendChild(li);
+    }
+  } catch (error) {
+    console.log(error);
+  }
+}
```

Comment:
Nice work implementing the functionalities of the application as required😉 but let's make use of the arrow function instead of a regular function😉 as an arrow function is less complicated and offers a lot of possibilities coupled with the fact that it is required for this [milestone](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md)👍
<img width="410" alt="image" src="https://user-images.githubusercontent.com/64643995/231777547-9e700f33-79aa-44b4-98aa-f903b65449ea.png">

Reply:
Hello @Ibizugbe,

Thanks for the feedback!

I'll make those changes right away 😊

Regards!

Reply:
**[REQUIRED CHANGES]**

- Hey, :wave: As suggested by the previous review [here](https://github.com/J-C-S-V/leaderboard-project/pull/2/files/bdbbf3b35bb0bb26628aae1bbdbde1f6a4159ee9#r1165543049), please, kindly implement the changes as you've not done anything yet. :pray: